<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QMainWindow" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnReservar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reservar Quarto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReservaDiversos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Reservar Diversos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Informacoes do Usuario </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Senha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Idade:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Cpf:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtNome">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtSenha">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtIdade">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtSexo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="txtCpf">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>191</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSair">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout </string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>191</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quartos Reservados:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Reservas Diversas:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExlcuir">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remover Reserva</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
